jsapp/js/projects/projectsTable/projectsTableHeader.module.scss

```diff
@@ -152,6 +168,7 @@ $resizer-width: sizes.$x16 + $line-width;
   opacity: 1;
 }
 // Always hide leftmost width indicator
-.columnRoot:first-child::before {
+// nth-child(2) because of checkbox column
+.columnRoot:nth-child(2)::before {
   display: none;
 }
```

Comment:
The first column header cell (empty one, above checkboxes) has pointer cursor visible when mouseover-ing. It's not clickable, so we should use the default cursor.